.eslintrc

```diff
@@ -9,8 +9,10 @@
     "consistent-return": "off",
     "no-restricted-syntax": "off",
     "import/prefer-default-export": "off",
-    "camelcase": "off"
-  },
+    "camelcase": "off",
+    "no-shadow": "off",
+    "@typescript-eslint/no-shadow": ["error"]
+    },
```

Comment:
These changes are not needed. You can simply `git checkout master .eslintrc` to reset the file to the original.